### DISCUSSION TITLE
VPN в СВОИ proxy

### DISCUSSION THREAD

**Discussion by Ronln** [anticensority/runet-censorship-bypass#204] [Ideas]
Можно ли прикрутить возможность добавления VPN  в категорию "Свои Прокси" вида user:pass@host для использования с белым списком?

**Comment by ilyaigpetrov** [comment]
VPN не поддерживается в браузерных расширениях совсем. Когда пишут о VPN-расширениях, то внутри всегда оказвыается обычный прокси. Если ваш VPN-провайдер помимо VPN предоставляет доступ по прокси, то их можно добавить в расширение, но поддерживаются прокси не всех типов: см. https://github.com/anticensority/runet-censorship-bypass/wiki/Прокси-и-пароль и https://github.com/anticensority/runet-censorship-bypass/wiki/Свои-прокси-в-расширении.
